index.html

```diff
@@ -33,9 +36,149 @@
+            <li class="label-project">Javascript</li>
+            <li class="label-project">CSS</li>
+            <li class="label-project">HTML</li>    
+           
+        </ul>
+           <button class="see-post"><p class="see-button">see project</p></button>
+        </div>
+        <div class="grid-item fourth">
+            <div class="item image"></div> 
+           <h2 class="title">Multi-Post Stories Gain + Glory</h2>
+          
+            <ul class="tagcontainer">
+            <li class="label-project">Ruby on Rails</li>
+            <li class="label-project">Javascript</li>
+            <li class="label-project">CSS</li>
+            <li class="label-project">HTML</li>
+            </ul>
+           
+           <button class="see-post"><p class="see-button">see project</p></button>
+        </div>
+        <div class="grid-item fiveth"><div class="item image"></div> 
+        <h2 class="title">Multi-Post Stories Gain + Glory</h2>
+       
+            <ul class="tagcontainer">
+         <li class="label-project">Ruby on Rails</li>
+         <li class="label-project">Javascript</li>
+         <li class="label-project">CSS</li>
+         <li class="label-project">HTML</li>
+            </ul>
+        
+        <button class="see-post"><p class="see-button">see project</p></button></div>
+        <div class="grid-item sixth">
+            <div class="item image"></div> 
+           <h2 class="title">Multi-Post Stories Gain + Glory</h2>
+           
+            <ul class="tagcontainer">
+            <li class="label-project">Ruby on Rails</li>
+            <li class="label-project">Javascript</li>
+            <li class="label-project">CSS</li>
+            <li class="label-project">HTML</li>
+            </ul>
+          
+           <button class="see-post"><p class="see-button">see project</p></button>
+        </div>
+        <div class="grid-item ">
+            <div class="item image"></div> 
+           <h2 class="title">Multi-Post Stories Gain + Glory</h2>
+           
+            <ul class="tagcontainer">
+            <li class="label-project">Ruby on Rails</li>
+            <li class="label-project">Javascript</li>
+            <li class="label-project">CSS</li>
+            <li class="label-project">HTML</li>
+            </ul>
+           
+           <button class="see-post"><p class="see-button">see project</p></button>
+        </div>
+    </div>
+</section>
+<section>
+   
+        <div class="about-me">
+            
+              <!-- <img class="right" src="./right.png" alt="image">   -->
+            <h2 class="about">About Me</h2>
+            
+            <p class="about-myself"> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                don’t hesitate to contact me.</p>
+               
+                <button class="get-my-resume"><p class="resume">Get My Resume</p></button>
```

Comment:
- [ ] Great job implementing the suggested button states, But I noticed you added a lot of CSS properties on the hover. I think you are only supposed to change the button background color when it's hovered and not other properties like the `text size, border, etc`. I'll suggest you implement the hover state to meet the given design. :+1: :100:  Below is how your button responds on hover and you can see that it doesn't give the best user experience.

- YOUR UI:

https://user-images.githubusercontent.com/107632780/235317013-81bfbbca-874b-44a4-b630-4d44b67b40ca.mp4